x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -784,7 +784,7 @@ export async function handleLoadMetadataTransformStats(http: HttpStart, store: E
 
   try {
     const transformStatsResponse: TransformStatsResponse = await http.get(
-      METADATA_TRANSFORM_STATS_URL
+      METADATA_TRANSFORMS_STATUS_ROUTE
```

Comment:
Do you know what happens if the API call returns a failure in the code below (line 795)? Do we show it on the UI?

Reply:
It shouldn't show up in the UI. the banner only shows if the response has transforms with the warning states we defined. so no transforms returned would just be silently ignored. though this should be exceedingly rare for permission based reasons since the banner lives in the endpoint list component which uses the same permissions.